unit-tests-jest/numbers.test.js

```diff
@@ -0,0 +1,59 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+describe('evenNumbers(numbers)', () => {
+  it('returns a new array with the even numbers only', () => {
+    const result1 = evenNumbers([0, 2, 3, 4, 5, 6, 7, 12, 14]);
+    expect(result1).toEqual([0, 2, 4, 6, 12, 14]);
+    const result2 = evenNumbers([]);
+    expect(result2).toEqual([]);
+    const result3 = evenNumbers([-8, -3]);
+    expect(result3).toEqual([-8]);
+  });
```

Comment:
It's best to create 3 small tests instead of having multiple calls to the function under test in the same test.
```
describe('evenNumbers(numbers)', () => {
  it('returns a new array with the even numbers only', () => {
    const result1 = evenNumbers([0, 2, 3, 4, 5, 6, 7, 12, 14]);
    expect(result1).toEqual([0, 2, 4, 6, 12, 14]);
  });
  it('returns empty array if empty', () => {
    const result2 = evenNumbers([]);
    expect(result2).toEqual([]);
  });
  it('works with negative numbers', () => {
    const result3 = evenNumbers([-8, -3]);
    expect(result3).toEqual([-8]);
  });
});
```